toast-tk-runtime/src/main/java/io/toast/tk/runtime/emailReport.java

```diff
@@ -0,0 +1,58 @@
+package io.toast.tk.runtime;
+
+import java.util.Properties;
+import javax.mail.Message;
+import javax.mail.MessagingException;
+import javax.mail.Session;
+import javax.mail.Transport;
+import javax.mail.internet.AddressException;
+import javax.mail.internet.InternetAddress;
+import javax.mail.internet.MimeMessage;
+
+/**
+ * @author Crunchify.com
+ * 
+ */
+
+public class emailReport {
+
+	static Properties mailServerProperties;
+	static Session getMailSession;
+	static MimeMessage generateMailMessage;
+
+	public static void main(String args[]) throws AddressException, MessagingException {
+		generateAndSendEmail();
+		System.out.println("\n\n ===> Your Java Program has just sent an Email successfully. Check your email..");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.talanlabs.com/coding_rules#rule_key=squid%3AS106)